ansible.yaml

```
@@ -0,0 +1,69 @@
+- name: Install app on new web server
+  hosts: managed
+  remote_user: ec2-user
+
+  vars_prompt:
+    - name: secret_key
+      prompt: What is the secret key?
+
+    - name: trello_api_key
+      prompt: What is the trello api key?
+    
+    - name: trello_api_secret
+      prompt: What is the trello api secret?
+    
+    - name: trello_api_token
+      prompt: What is the trello api token?
+
+  tasks:
+    - name: Install Git
+      yum:
+        name: git
+        state: present
+      become: yes
+
+    - name: Install Python3
+      yum:
+        name: python3
+        state: present
+      become: yes
```

Minor, but worth noting that we can combine these if we choose:
```suggestion
    - name: Install Git & Python3
      yum:
        name: 
          - git
          - python3
        state: present
      become: yes
```